assembly/src/errors.rs

```diff
@@ -135,7 +142,7 @@ impl fmt::Display for AssemblyError {
             DuplicateProcName(proc_name, module_path) => write!(f, "duplicate proc name '{proc_name}' in module {module_path}"),
             DuplicateProcId(proc_id) => write!(f, "duplicate proc id {proc_id}"),
             ExportedProcInProgram(proc_name) => write!(f, "exported procedure '{proc_name}' in executable program"),
-            ImportedProcModuleNotFound(proc_id) => write!(f, "module for imported procedure {proc_id} not found"),
+            ImportedProcModuleNotFound(proc_id, proc_name) => write!(f, "module for imported procedure {proc_name} with ID {proc_id} not found"),
```

Comment:
minor nit: let's put backticks around procedure name - the error message will look better this way.
```
module for imported procedure `{proc_name}` with ID {proc_id} not found
```